pibooth/view/background.py

```diff
@@ -195,22 +196,24 @@ def paint(self, screen):
 
 class IntroBackground(Background):
 
-    def __init__(self, arrow_location=ARROW_BOTTOM, arrow_offset=0):
+    def __init__(self, arrow_location=ARROW_BOTTOM, arrow_offset=0, full_width=True):
```

Comment:
The full_width option is only used for arrow_location=ARROW_TOUCH and no print available, I guess we can find a way to avoid using a new param for that.

Reply:
Ok I will try to remove the parameter